### PR TITLE
Add concept page for CLA assistant

### PR DESCRIPTION
**Pull request by jdbaldry**
Explain the purpose for the CLA assistant and how to trigger the workflow if it does not report automatically.

Signed-off-by: Jack Baldry <jack.baldry@grafana.com>

### CODE REVIEW

File: docs/sources/review/cla-assistant/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+You can request the CLA assistant to run again by browsing to the check URL for the PR.
+
+<!-- vale Grafana.WordList = YES -->
+
+The format of the URL is `https://cla-assistant.io/check/grafana/<REPOSITORY>?pullRequest=<PR>`.
+
+- _`REPOSITORY`_ is the GitHub repository name.
+- _`PR`_ is the Pull Request number in the repository.
+
+To trigger the CLA assistant workflow for the Pull Request `https://github.com/grafana/grafana/pull/1`, the URL is `https://cla-assistant.io/check/grafana/agent?pullRequest=1`.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:37]:
```suggestion
For example, to trigger the CLA assistant workflow for the pull request `https://github.com/grafana/grafana/pull/1`, the URL is `https://cla-assistant.io/check/grafana/agent?pullRequest=1`.
```

Comment (clayton-cornell) [inline docs/sources/review/cla-assistant/index.md:37]:
```suggestion
To trigger the CLA assistant workflow for the Pull Request `https://github.com/grafana/grafana/pull/1`, the URL is `https://cla-assistant.io/check/grafana/grafana?pullRequest=1`.
```

Wouldn't the example URL here be the Grafanb repo?

File: docs/sources/review/cla-assistant/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+In such a circumstance, wait a few hours and try again later.
+{{% /admonition %}}
+
+## Waiting for status to be reported
+
+<!-- vale Grafana.WordList = NO -->
+<!-- "check" here is used as a noun not a verb. TODO: update the WordList to only lint "check" as a verb. -->
+
+Occasionally, the CLA assistant workflow is not reported on an open PR, preventing the PR from being merged.
+You can request the CLA assistant to run again by browsing to the check URL for the PR.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:28]:
```suggestion
You can request that the CLA assistant run again by browsing to the check URL for the PR.
```

File: docs/sources/review/cla-assistant/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
+---
+title: About CLA assistant
+description: Understand the CLA assistant and its use on Github Pull Requests.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:3]:
```suggestion
description: Understand the CLA assistant and its use on GitHub pull requests.
```

Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:3]: GitHub should always be spelled with a capital H and pull requests never needs to be capitalized.

```suggestion
description: Understand the CLA assistant and its use on GitHub pull requests.
```

File: docs/sources/review/cla-assistant/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+---
+
+# About CLA assistant
+
+[CLA assistant](https://cla-assistant.io/) is used by Grafana Labs to ensure that all contributors to a GitHub Pull Request (PR) have signed the Grafana Labs Contributor License Agreement (CLA).
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:8]:
```suggestion
[CLA assistant](https://cla-assistant.io/) is used by Grafana Labs to ensure that all contributors to a GitHub pull request (PR) have signed the Grafana Labs Contributor License Agreement (CLA).
```

File: docs/sources/review/cla-assistant/index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+Grafana Labs does not operate the CLA assistant infrastructure, instead it is provided by SAP.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:10]:
```suggestion
Grafana Labs doesn't operate the CLA assistant infrastructure; instead it's provided by SAP.
```

File: docs/sources/review/cla-assistant/index.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+The CLA assistant project is open source and the source code can be found in the [cla-assistant repository](https://github.com/cla-assistant/cla-assistant).
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:11]:
```suggestion
The CLA assistant project is open source and you can find the source code in the [cla-assistant repository](https://github.com/cla-assistant/cla-assistant).
```

File: docs/sources/review/cla-assistant/index.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+
+The workflow runs automatically on all PRs.
+During periods of high load on the CLA assistant infrastructure, the workflow can fail to report a status.
+If it does not report back and all other Continuous Integration (CI) checks have completed, refer to [Waiting for status to be reported](#waiting-for-status-to-be-reported) to rerun the workflow.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:15]:
```suggestion
If it doesn't report back and all other continuous integration (CI) checks have completed, follow the directions in [Waiting for status to be reported](#waiting-for-status-to-be-reported) to rerun the workflow.
```

File: docs/sources/review/cla-assistant/index.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+
+{{% admonition type="note" %}}
+During prolonged periods of high load on the CLA assistant infrastructure, even rerunning the workflow doesn't result in a successful status.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:18]:
```suggestion
During prolonged periods of high load on the CLA assistant infrastructure, even rerunning the workflow might not result in a successful status.
```

File: docs/sources/review/cla-assistant/index.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+In such a circumstance, wait a few hours and try again later.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:19]:
```suggestion
In this case, wait a few hours and try again later.
```

File: docs/sources/review/cla-assistant/index.md [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+{{% /admonition %}}
+
+## Waiting for status to be reported
+
+<!-- vale Grafana.WordList = NO -->
+<!-- "check" here is used as a noun not a verb. TODO: update the WordList to only lint "check" as a verb. -->
+
+Occasionally, the CLA assistant workflow is not reported on an open PR, preventing the PR from being merged.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:27]:
```suggestion
Occasionally, the CLA assistant workflow isn't reported on an open PR, preventing the PR from being merged.
```

File: docs/sources/review/cla-assistant/index.md (hunk 2 above)
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:28]:
```suggestion
You can request that the CLA assistant run again by browsing to the check URL for the PR.
```

File: docs/sources/review/cla-assistant/index.md [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+You can request the CLA assistant to run again by browsing to the check URL for the PR.
+
+<!-- vale Grafana.WordList = YES -->
+
+The format of the URL is `https://cla-assistant.io/check/grafana/<REPOSITORY>?pullRequest=<PR>`.
+
+- _`REPOSITORY`_ is the GitHub repository name.
+- _`PR`_ is the Pull Request number in the repository.
```
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:35]:
```suggestion
- _`PR`_ is the pull request number in the repository.
```

File: docs/sources/review/cla-assistant/index.md (hunk 1 above)
Comment (imatwawana) [inline docs/sources/review/cla-assistant/index.md:37]:
```suggestion
For example, to trigger the CLA assistant workflow for the pull request `https://github.com/grafana/grafana/pull/1`, the URL is `https://cla-assistant.io/check/grafana/agent?pullRequest=1`.
```

**Review by imatwawana — APPROVED** [review]
Made a bunch of wording edits for style adherence and clarity, but otherwise looks good. Thanks for adding!

File: docs/sources/review/cla-assistant/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/review/cla-assistant/index.md]: Good catch, thank you. That's fixed in [ea99cb9](https://github.com/grafana/writers-toolkit/pull/302/commits/ea99cb95c3452dd786cf046a369a70aee5053060).

**Pull request merged**